tests/unit_tests.py

```diff
@@ -9,8 +9,10 @@
 sys.path.append(os.path.abspath("../"))
 from smartmoneyconcepts.smc import smc
 
-# import test data
-df = pd.read_csv("test_data/EURUSD_15M.csv")
+# define and import test data
+test_instrument = "EURUSD"
+instrument_data = f"{test_instrument}_15M.csv"
+df = pd.read_csv(os.path.join("test_data", test_instrument, instrument_data))
```

Comment:
I believe this can be adjusted slightly.
I believe the folder containing the test data should have the title "EURUSD_15M"
and the test data should be renamed from "EURUSD_15M.csv" to "data.csv"

this is because the contents of the folder will always be relevant to the same instument and timeframe